### PR TITLE
add http host to page_views

### PR DESCRIPTION
**Pull request by myneid**
add the http host for the use case of having multiple domains on the same codebase

### CODE REVIEW

**Comment by andreaselia** [comment]
@myneid thanks for the PR, StyleCI is failing though.

File: database/migrations/2024_03_22_100000_add_host_to_page_views.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('page_views', function (Blueprint $table) {
+            $table->dropColumn([
+                'host',
+
```
Comment (andreaselia) [inline database/migrations/2024_03_22_100000_add_host_to_page_views.php:32]: Remove blank line.

File: src/Http/Middleware/Analytics.php [hunk 2]
```diff
@@ -74,6 +74,7 @@ public function handle(Request $request, Closure $next)
             'country' => $agent->languages()[0] ?? 'en-en',
             'browser' => $agent->browser() ?? null,
             'device'  => $agent->deviceType(),
+            'host' => $request->getHost(),
```
Comment (andreaselia) [inline src/Http/Middleware/Analytics.php:77]: Match the arrow to the others.

**Review by andreaselia — CHANGES_REQUESTED** [review]

**Review by andreaselia — APPROVED** [review]

**Comment by andreaselia** [comment]
Thanks for your PR @myneid 👍

**Comment by myneid** [comment]
sorry, i thought i had used intellesense formatting automatically. meant to get back to this. thank you for this great package

**Comment by andreaselia** [comment]
All good mate, thought I'd just quickly sort it out so others can make use of it too.

You're welcome, and thank you for contributing to make it an even greater package 💪

**Pull request merged**